test/midje/emission/t_clojure_test_facade.clj

```diff
@@ -33,10 +33,9 @@
   (fact
     (:test result) => 2
     (:fail result) => 1
-    (nth (:lines result) 1) => #"FAIL in.*a-clojure-test-fail"
-    (nth (:lines result) 2) => #"expected"
-    (nth (:lines result) 3) => #"actual"
```

Comment:
removing the checks to `expected` and `actual` lines because they seem to differ between dev environments and what passes on my machine doesn't pass on travis